Failed to search with `422 Unprocessable Entity`

Failed to search when a keyword has `:`.

```
% ghkw -d language:ruby
2018/02/07 21:12:08 [DEBUG] Run as DEBUG mode
2018/02/07 21:12:08 [DEBUG] keyword: [language:ruby]
422 Unprocessable Entity
&{%!s(*http.bodyEOFSignal=&{0xc4203d4940 {0 0} true 0xc420046060 <nil> 0x11fbc70}) %!s(func()=<nil>)}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1291e67]

goroutine 22 [running]:
main.(*Searcher).searchRequest(0xc420244420, 0x7ffeefbff728, 0xd, 0xc42007c1e0)
        /Users/katsuhiko.yoshida/go/src/github.com/kyoshidajp/ghkw/cli.go:134 +0x2a7
created by main.(*Searcher).search
        /Users/katsuhiko.yoshida/go/src/github.com/kyoshidajp/ghkw/cli.go:144 +0xbf
```

Run successfully without `:`.

```
% ghkw -d languageruby
2018/02/07 21:13:07 [DEBUG] Run as DEBUG mode
2018/02/07 21:13:07 [DEBUG] keyword: [languageruby]
2018/02/07 21:13:09 [DEBUG] Keyword: languageruby (32)
| RANK |   KEYWORD    | TOTAL |
|------|--------------|-------|
|    1 | languageruby |    32 |
```

According to [GitHub Search Document](https://help.github.com/articles/searching-code/#considerations-for-code-search), Search query can't include ``. , : ; / \ ` ' " = * ! ? # $ & + ^ | ~ < > ( ) { } [ ]``.

--- a/cli.go
+++ b/cli.go
@@ -9,6 +9,7 @@ import (
 	"os"
 	"path/filepath"
 	"sort"
+	"strings"
 	"time"
 
 	"golang.org/x/oauth2"
@@ -36,6 +37,9 @@ const (
 	ExitCodeParseFlagsError
 	ExitCodeBadArgs
 )
+
+// Ignore char in keyword
+const IGNORE_KEYWORD_CHAR = ":"
 
 // Debugf prints debug output when EnvDebug is given
 func Debugf(format string, args ...interface{}) {
@@ -273,6 +277,10 @@ func getAccessToken() (string, error) {
 	return token, nil
 }
 
+func sanitizeKeyword(keyword string) string {
+	return strings.Replace(keyword, IGNORE_KEYWORD_CHAR, "", -1)
+}
+
 // NewClient creates SearchClient
 func NewClient(keywords []string, searchTerm SearchTerm) (*Searcher, error) {
 	token, err := getAccessToken()
@@ -290,7 +298,7 @@ func NewClient(keywords []string, searchTerm SearchTerm) (*Searcher, error) {
 
 	keywordsWithTotal := map[string]int{}
 	for i := range keywords {
-		keyword := keywords[i]
+		keyword := sanitizeKeyword(keywords[i])
 		keywordsWithTotal[keyword] = 0
 	}
 

